Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -984,9 +988,14 @@ - (void)viewWillDisappear:(BOOL)animated
 
     [super viewWillDisappear:animated];
 
-    self.isViewVisible = NO;
-
     [self.inputToolbar viewWillDisappear:animated];
+}
+
+- (void)viewDidDisappear:(BOOL)animated
```

Comment:
viewWillDisappear is called even if the transition away is cancelled. We should delay all possible teardown until viewDidDisappear.

Reply:
Maybe add a comment in `viewWillDisappear` noting that we shouldn't add logic there?

Reply:
Good point. Done!